### DISCUSSION TITLE
Reading the coverage report

### DISCUSSION THREAD

**Discussion by matthewcc** [ArtiomTr/jest-coverage-report-action#307] [Q&A]
Is it possible to read the output/result of the coverage comparison in a subsequent GHA step? I would like to have a step that reports a failure status if there was a decline in code coverage in a PR, as reported by this action.

**Comment by ArtiomTr** [comment]
Hello @matthewcc :wave:,

No, currently there is no possibility to get coverage comparison information, as it is an internal API.

**Reply by matthewcc** [reply]
Ok, thanks for the quick reply!
